src/components/common/logoImage/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { StyledImage } from './styled';
```

Comment:
The same

Reply:
Fix: Logo icon svg imported & used as ReactComponent